src/components/PackageList/PackageList.tsx

```diff
@@ -13,16 +13,16 @@ interface Props {
 }
 
 export const PackageList: React.FC<Props> = props => {
-  const renderPackages: () => React.ReactElement<HTMLElement>[] = () => {
+  const renderPackages: () => ReactElement<HTMLElement>[] = () => {
```

Comment:
Hm...I don't know if I agree with this type... maybe React.ReactNode?

Reply:
done